2-mandatory/4-people-i-know.js

```diff
@@ -394,15 +394,34 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
-/*
-5) Find "Multi-tasking" colleagues
+for (let i=0; i< friends.length; i++) {
+          for (let j=0; j < friends[i].colleagues.length; j++) {
+              if (friends[i].colleagues[j].name === "Stacie Villarreal") {
+                friendsWhoAreColleaguesOfStacie.push(
+                  `${friends[i].name.first} ${friends[i].name.last}`
+                );
+              }
+          }
+      }
+/*5) Find "Multi-tasking" colleagues
 Next, I want you to find all of the colleagues of my friends who are good at "Multi-tasking"
 You can tell if they are good at "Multi-tasking" because they will have it listed in their skills
 This time, I only want the full names of the people who can multitask
 */
 
-let colleaguesWhoCanMultitask = [];
-
+let arrayOfNames = [];
+for (let i = 0; i < friends.length; i++) {
+  for (let j = 0; j < friends[i].colleagues.length; j++) {
+    for (let k = 0; k < friends[i].colleagues[j].skills.length; k++) {
+      if (friends[i].colleagues[j].skills.includes("Multi-tasking")) {
```

Comment:
Is using the 'k' loop here needed? as on the next line you are using `includes`
You may also want to consider using `for of` loops which can help with readability